### DISCUSSION TITLE
dbbackup with django/docker/celery/celery-beat not working: no error but no file stored?

### DISCUSSION THREAD

**Discussion by LECARROU** [Archmonger/django-dbbackup#583] [Q&A] [answered]
# Bug Report

Try to implement a periodic postgresql database backup using django/docker/celelry/celery-beat but can not find my file

## Describe the bug

I can create 'manually' a dbbackup file using python manage.py dbbackup and can recover the backup file default-PRT-063-2020-12-29-174901.dump in backup folder as specified in settings

but when I try to use celery and call_command('dbbackup'), seems to works as I have no error (see below) 
but I could not find the file anywhere (locally or in my docker container)

I notice that file extension is not the same .dump (manually) and .psql (call_command)

below are trace and all seems to be OK

celery_1       | Backing Up Database: db_dev
celery_1       | [2020-12-29 16:47:00,103: INFO/ForkPoolWorker-4] Backing Up Database: db_dev
celery_1       | Writing file to default-682c25120a03-2020-12-29-164700.psql
celery_1       | [2020-12-29 16:47:01,258: INFO/ForkPoolWorker-4] Writing file to default-682c25120a03-2020-12-29-164700.psql
celery_1       | [2020-12-29 16:47:01,263: INFO/ForkPoolWorker-4] Task cafe.tasks.backup[046e9740-a4e2-49dd-9ab6-b5d5a7305ae7] succeeded in 1.1994489999997313s: 'Backed up successfully: 2020-12-29 16:47:01.260865+00:00'

I have been trying for 2 day and nothing work
hope someone can help me

### Django-dbbackup
- pypi: 3.3.0

### External tools

- Python: 3.7.4
- Django: 2.2.5
- OS: Windows
- Celery et Celery-beat
- Docker et docker-compose

**Comment by d-kahara** [comment, marked as answer]
If you have a separate container for your celery workers, the backup files are most likely being dumped in there.
